### ISSUE TITLE
Split across soft as well as hard line breaks

### ISSUE THREAD

**Issue by davestewart** [Wolfr/textcutter#2]
Hello,

As per my comment on the [plugin page](https://www.figma.com/community/plugin/739131137116544548/textcutter), you should probably split on all line breaks.

Change [this line](https://github.com/Obra-Studio/textcutter/blob/master/textcutter-code/code.ts#L78
) to:

```js
const lines = node.characters.split(/[\n\r]/g)
```

Cheers!

**Comment by Wolfr** [comment]
Thanks Dave, the line you referenced is about the remove bullets part.

For the split command, I ended up doing some testing, and I was already catching soft line breaks before. What I am now also catching is the U+2028 character, which sometimes seems to be used for line breaks when pasting from IDEs. I can see it showing up when I paste from IntelliJ Ultimate, for example.

I updated the plugin based on your comment. Thanks for that!

Might I ask which IDE you are using?

**Comment by davestewart** [comment]
Hey there. Webstorm.

**Comment by Wolfr** [comment]
@davestewart Thanks! So that's a similar one tech-wise as I was testing with (IntelliJ is based on the same engine I think)

**Comment by davestewart** [comment]
Yeah, should be the same! It's free for non commercial work if you need to test it directly

**Comment by gavinmcfarland** [comment]
This should be fixed in this pull request #3. I believe it was happening due to different platforms or older formats. 


Format | Escape Sequence | Description
-- | -- | --
LF | \n | Unix, macOS (modern)
CRLF | \r\n | Windows
CR | \r | Classic Mac OS
Unicode LS | \u2028 | Line Separator
Unicode PS | \u2029 | Paragraph Separator
Mixed | \n\r | Rare, malformed or legacy files
